vlinder/tests/pages/homepage.test.js

```diff
@@ -3,34 +3,40 @@ import { render, screen, fireEvent, waitFor, user } from '@testing-library/react
 import HomePage from '@/app/homepage/page';
 import { createClient } from '@/utils/supabase/client';
 
-import '@testing-library/jest-dom';
+import { useRouter } from 'next/router';
 
 
 const supabase = createClient();
 export default supabase;
 
+jest.mock("next/navigation", () => ({
+    useRouter() {
+      return {
+        prefetch: () => null
+      };
+    }
+  }));
 
 describe('HomePage', () => {
-
     it('renders loading state initially', () => {
-        render(<HomePage />);
+        render(<HomePage/>);
         expect(screen.getByText('Loading...')).toBeInTheDocument();
     });
 
-    it('renders profiles after loading', async () => {
```

Comment:
commants can be removed? for develpment?

Reply:
no